packages/Python/lldbsuite/test/lang/swift/clangimporter/bridging_header_headermap/Makefile

```diff
@@ -16,35 +16,11 @@ all: a.out
 
 include $(LEVEL)/Makefile.rules
 
-.PHONY: libDylib
-
-a.out: main.swift libDylib
-#       Avoid building a dSYM for the main application.
-	$(SWIFT_FE) -c -primary-file $< $(SWIFT_FEFLAGS) \
-	  -module-name main -o main.o \
-          -emit-module-path main.part.swiftmodule \
-	  -import-objc-header $(SRCDIR)/bridging-header.h \
-          -Xcc -I$(SRCDIR)/foo.hmap -I. -I$(SRCDIR)
-	$(SWIFT_FE) $(SWIFT_FEFLAGS) \
-	  -merge-modules -emit-module main.part.swiftmodule \
-	  -parse-as-library -sil-merge-partial-modules \
-	  -disable-diagnostic-passes -disable-sil-perf-optzns \
-	  -import-objc-header $(SRCDIR)/bridging-header.h \
-          -Xcc -I$(SRCDIR)/foo.hmap \
-          -module-name main -o main.swiftmodule -I$(SRCDIR)
-	$(SWIFTC) -o $@ main.o -Xlinker -add_ast_path -Xlinker main.swiftmodule \
-          -lDylib -L.
-ifneq "$(CODESIGN)" ""
-	$(CODESIGN) -s - "$@"
-endif
-	$(DSYMUTIL) a.out
-
-libDylib: dylib.swift
-	$(SWIFTC) -g -Onone $^ -emit-library -module-name Dylib -emit-module \
-	  -Xlinker -install_name -Xlinker @executable_path/$@.dylib $(SWIFTFLAGS)
-ifneq "$(CODESIGN)" ""
-	$(CODESIGN) -s - "$@"
-endif
+libdylib.dylib: dylib.swift
```

Comment:
Why hard code the `dylib` extension?  The bridging header stuff actually does work on other targets

Reply:
Just FYI, buck does generate headermaps as well (and they are used pretty extensively elsewhere as well)

Reply:
I was happier before I knew that.